data-prepper-core/src/main/java/com/amazon/dataprepper/DataPrepper.java

```diff
@@ -27,32 +32,23 @@
 public class DataPrepper {
     private static final Logger LOG = LoggerFactory.getLogger(DataPrepper.class);
 
-    private static final PrometheusMeterRegistry sysJVMMeterRegistry = new PrometheusMeterRegistry(PrometheusConfig.DEFAULT);
+    private static final CompositeMeterRegistry systemMeterRegistry = new CompositeMeterRegistry();
 
     private Map<String, Pipeline> transformationPipelines;
 
     private static volatile DataPrepper dataPrepper;
 
     private static DataPrepperServer dataPrepperServer;
-    private static DataPrepperConfiguration configuration = DataPrepperConfiguration.DEFAULT_CONFIG;
-
-    static {
-        new ClassLoaderMetrics().bindTo(sysJVMMeterRegistry);
-        new JvmMemoryMetrics().bindTo(sysJVMMeterRegistry);
-        new JvmGcMetrics().bindTo(sysJVMMeterRegistry);
-        new ProcessorMetrics().bindTo(sysJVMMeterRegistry);
-        new JvmThreadMetrics().bindTo(sysJVMMeterRegistry);
-    }
+    private static DataPrepperConfiguration configuration;
 
     /**
-     * Set the DataPrepperConfiguration from a file
+     * Set the DataPrepperConfiguration from file, if exists; else use default configuration
      * @param configurationFile File containing DataPrepperConfiguration yaml
      */
-    public static void configure(final String configurationFile) {
-        final DataPrepperConfiguration dataPrepperConfiguration =
+    public static void configure(@Nullable final String configurationFile) {
+        configuration = configurationFile == null ? DataPrepperConfiguration.DEFAULT_CONFIG :
```

Comment:
I thought about the Optional<String> but then the suggestion is not to use Optional as parameters/arguments. I will attempt to add parameter-less `configureWithDefaults()`

Reply:
Good point - I shouldn't have suggested an Optional as a parameter. It's better to avoid method behavior splitting based on null/non-null args if possible.